tools/site-creator/create-site.js

```diff
@@ -224,16 +232,21 @@ async function updateDaConfig(data) {
   if (!updateRes.ok) { throw new Error(`Failed to update config: ${updateRes.statusText}`); }
 }
 
-function checkAuth() {
+function verifyAuthentication() {
   if (!token || token === 'undefined') {
     throw new Error('Please sign in.');
```

Comment:
And I suppose if the token is set but invalid, they'll just get an error at the next step. LGTM

Reply:
Yes, I think this is correct. I did not test, so we are relying purely on the API response from DA in that case. But it should disallow.